packages/util/src/constants.ts

```diff
@@ -49,6 +49,27 @@ export enum Network {
   optimism = 'optimism',
 }
 
+export const PointsSeasonsData = {
```

Comment:
Can you put this in the points dashboard folder instead? This is global to everything